test/files.js

```diff
@@ -0,0 +1,167 @@
+/* eslint-env mocha */
+'use strict'
+
+const expect = require('chai').expect
+const DAGNode = require('ipfs-merkle-dag').DAGNode
+const bs58 = require('bs58')
+const bl = require('bl')
+const Readable = require('readable-stream')
+const path = require('path')
+const isNode = require('detect-node')
+
+module.exports = (common) => {
+  describe('.files/add', () => {
+    let testfile
+    let testfileBig
+    let ipfs
+
+    before((done) => {
+      // load test data
+      if (isNode) {
+        const fs = require('fs')
+
+        const testfilePath = path.join(__dirname, './data/testfile.txt')
+        testfile = fs.readFileSync(testfilePath)
+
+        const testfileBigPath = path.join(__dirname, './data/15mb.random')
+        testfileBig = fs.createReadStream(testfileBigPath, { bufferSize: 128 })
+      } else {
+        testfile = require('raw!./data/testfile.txt')
```

Comment:
We now use the browserify transform to support fs.readFileSync on the transpilation process as well. We don't need to use the `require('raw`